Observing-Interface-Setup/environments.rst

```diff
@@ -0,0 +1,174 @@
+
+- Successfully access nublado and other observing interface tools.
+
+.. _Observing-Interface-Environments-Procedure-Steps:
+
+Procedure Steps
+===============
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+In a browser open the links specified above to the environment you want to interact with.
+
+For instance, open the nublado link in :ref:`NTS <Observing-Interface-Environments-NTS>` to interact with the observatory control system in that environment.
+
+.. _Observing-Interface-Environments-NTS:
+
+NCSA Test Stand (NTS)
+---------------------
+
+The NTS is currently being used as a staging platform to test software deployment prior to deployment at the summit.
+Systems that control hardware at the :ref:`Summit <Observing-Interface-Environments-Summit>` run here in simulation mode.
+Once deployment is tested here, the platform is made available for general users for training, testing and developing of control algorithm and procedures.
+
+- Nublado: https://lsst-nts-k8s.ncsa.illinois.edu/
+- Chronograf: https://lsst-chronograf-nts-efd.ncsa.illinois.edu/
+- LOVE:
+    .. prompt:: bash
+
+       ssh -L 8080:lsst-teststand-ts1.ncsa.illinois.edu:80 -J lsst-login03.ncsa.illinois.edu lsst-teststand-ts1.ncsa.illinois.edu
+
+    - Use your NCSA account to login when prompted.
+    - You will be prompted for your password twice after running the command above.
+    - Once logged in, open http://localhost:8080/ on a browser.
+
+- ArgoCD: https://lsst-argocd-nts-efd.ncsa.illinois.edu/
+- Slack: ncsa-integ-teststand
+
+
+.. _Observing-Interface-Environments-BTS:
+
+Base Test Stand (BTS)
+---------------------
+
+BTS is a simulation environment running at the base facility in La Serena.
+The main difference between the :ref:`NTS <Observing-Interface-Environments-NTS>` and this environment is that some systems will run using hardware simulators.
+
+Hardware simulators are, in general, higher fidelity than those of pure software available at :ref:`NTS <Observing-Interface-Environments-NTS>`.
+For that, BTS will be used mostly for higher fidelity testing, development and debugging.
```

Comment:
maybe?

Reply:
Maybe just add a sentence saying that this teststand will be particularily useful for testing of lower-level hardware interfaces because of the presence of the simulators.